internal/operators/common/consts.go

```diff
@@ -0,0 +1,6 @@
+package common
+
+const (
+	NodeHealthcheckOperatorName     = "node-healthcheck"
+	SelfNodeRemediationOperatorName = "self-node-remediation"
```

Comment:
Why are these constants needed? Can't you import the corresponding operator and use the name from there?

Reply:
Cyclic imports